package.json

```diff
@@ -29,17 +28,10 @@
     "url": "https://github.com/seegno/jscs-config-seegno/issues"
   },
   "homepage": "https://github.com/seegno/jscs-config-seegno#readme",
-  "dependencies": {
-    "jscs": "^2.5.0"
-  },
   "devDependencies": {
-    "babel": "^5.8.23",
+    "jscs": "^2.8.0",
     "mocha": "^2.3.3",
-    "should": "^7.1.0"
-  },
-  "peerDependencies": {
-    "jscs": ">= 2.5",
```

Comment:
I guess we can still keep it as part of the installation command at least. Let's move on.